src/components/inputs/localized-multiline-text-input/localized-multiline-text-input.js

```diff
@@ -115,9 +117,14 @@ export class LocalizedMultilineTextInput extends React.Component {
 
   expandAllTranslations = () =>
     this.setState(prevState => ({
-      expandedTranslations: mapValues(
-        prevState.expandedTranslations,
-        () => true
+      expandedTranslations: Object.keys(prevState.expandedTranslations).reduce(
```

Comment:
You could also just dump this into a `utils` in UIKit itself as a `mapValues` then the code would be even shared.